airbyte-scheduler/src/main/java/io/airbyte/scheduler/JobSubmitter.java

```diff
@@ -91,146 +84,11 @@ public void run() {
-    // build deterministic job and attempt uuids based off of the scope,which should be unique across
-    // all instances of airbyte installed everywhere).
-    final UUID jobUuid = UUID.nameUUIDFromBytes((job.getScope() + job.getId() + job.getAttempts()).getBytes(Charsets.UTF_8));
-    final UUID attemptUuid = UUID.nameUUIDFromBytes((job.getScope() + job.getId() + job.getAttempts()).getBytes(Charsets.UTF_8));
-    metadata.put("job_uuid", jobUuid);
-    metadata.put("attempt_uuid", attemptUuid);
-
-    switch (job.getConfig().getConfigType()) {
-      case CHECK_CONNECTION_SOURCE, DISCOVER_SCHEMA -> {
-        final StandardSourceDefinition sourceDefinition = configRepository.getSourceDefinitionFromSource(UUID.fromString(job.getScope()));
-
-        metadata.put("connector_source", sourceDefinition.getName());
-        metadata.put("connector_source_definition_id", sourceDefinition.getSourceDefinitionId());
-      }
-      case CHECK_CONNECTION_DESTINATION -> {
-        final StandardDestinationDefinition destinationDefinition = configRepository
-            .getDestinationDefinitionFromDestination(UUID.fromString(job.getScope()));
-
-        metadata.put("connector_destination", destinationDefinition.getName());
-        metadata.put("connector_destination_definition_id", destinationDefinition.getDestinationDefinitionId());
-      }
-      case GET_SPEC -> {
-        // no op because this will be noisy as heck.
-      }
-      case SYNC -> {
-        final UUID connectionId = UUID.fromString(job.getScope());
-        final StandardSyncSchedule schedule = configRepository.getStandardSyncSchedule(connectionId);
-        final StandardSourceDefinition sourceDefinition = configRepository
-            .getSourceDefinitionFromConnection(connectionId);
-        final StandardDestinationDefinition destinationDefinition = configRepository
-            .getDestinationDefinitionFromConnection(connectionId);
-
-        metadata.put("connection_id", connectionId);
-        metadata.put("connector_source", sourceDefinition.getName());
-        metadata.put("connector_source_definition_id", sourceDefinition.getSourceDefinitionId());
-        metadata.put("connector_destination", destinationDefinition.getName());
-        metadata.put("connector_destination_definition_id", destinationDefinition.getDestinationDefinitionId());
-
-        String frequencyString;
-        if (schedule.getManual()) {
-          frequencyString = "manual";
-        } else {
-          final long intervalInMinutes = TimeUnit.SECONDS.toMinutes(ScheduleHelpers.getIntervalInSecond(schedule.getSchedule()));
-          frequencyString = intervalInMinutes + " min";
-        }
-        metadata.put("frequency", frequencyString);
-      }
-    }
-    return metadata;
+    // temporal submit.
+    // not a great story on how we recover from this submission failing...
+    final UUID temporalId = temporalClient.submit(job); // maybe need to all submit info from workerrunfactory.
```

Comment:
`temporalClient` is whatever that client temporal gives us. obviously will need a bit of massaging.